src/main/java/io/cdap/plugin/salesforce/SalesforceSchemaUtil.java

```diff
@@ -79,7 +81,38 @@ public static Schema getSchema(AuthenticatorCredentials credentials, SObjectDesc
     SObjectsDescribeResult describeResult = new SObjectsDescribeResult(partnerConnection,
       sObjectDescriptor.getAllParentObjects());
 
-    return getSchemaWithFields(sObjectDescriptor, describeResult);
+    return getSchemaWithFields(sObjectDescriptor, describeResult, "output");
+  }
+
+  /**
+   * Generates CDAP schema for multiple SObjects, where each field of type record represents one SObjects.
```

Comment:
Is this only used to generate CDAP schema in the multiple SObject case? If so imo the function name should reflect that